src/app/sidenav/sidenav.component.scss

```diff
@@ -0,0 +1,15 @@
+::ng-deep .p-menu{
+  width: 15%;
+  position: fixed;
```

Comment:
@Sukanya0816 Any reason why you used fixed position here.

Reply:
if i give postion fixed only it is taking the height 100% otherwise it is taking content height.

Reply:
We should give fixed height for header and remove that height from content height. Try to use calc function and 100vh.
Example header height is 80px then content div height would be height=calc(100vh - 80px)

Reply:
okay, I got it thank you